test/apps/export-webpack/src/client.js

```diff
@@ -0,0 +1,9 @@
+import * as sapper from '@sapper/app';
```

Comment:
I copy-pasta'd the `test/apps/export` project into `test/apps/export-webpack`, changing only `rollup.config.js` by replacing it with `webpack.config.js`.